src/index.js

```diff
@@ -1,16 +1,16 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 import { View, ActivityIndicator, StyleSheet } from 'react-native';
 import { withGoogleMap, GoogleMap } from 'react-google-maps';
 import Marker from './Marker';
 
 const GoogleMapContainer = withGoogleMap(props => <GoogleMap {...props} ref={props.handleMapMounted} />);
 
-class MapView extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { center: { lat: props.region.latitude, lng: props.region.longitude } };
-  }
-
+function calcZoom(region) {
+  if (!region.latitudeDelta || !region.longitudeDelta) return 13
```

Comment:
does 13 correspond to the default zoom of react-native-maps?

Reply:
Yes, exactly